readme.md

```diff
@@ -69,17 +63,13 @@ Once installation is working, try creating and requesting resources. Note that t
 
 * **Question:** Describe what the above code is adding to your schema.
 
-* **Your Answer:**
-
----
+* **Your Answer:** Adding a unique id for each entry.
```

Comment:
This is not quite right. If you include the above in your code, it will require you to send an ObjectID when creating a new record.